src/config.rs

```diff
@@ -31,17 +31,31 @@ pub struct Config {
     cert_file: Option<String>,
     #[clap(long)]
     /// Path to admin.macaroon file for lnd
-    macaroon_file: Option<String>,
+    admin_macaroon_file: Option<String>,
+    #[clap(long)]
+    /// Path to invoice.macaroon file for lnd
+    invoice_macaroon_file: Option<String>,
     #[clap(long)]
     /// Include route hints in invoices
     pub route_hints: bool,
 }
 
 impl Config {
     pub fn macaroon_file(&self) -> String {
-        self.macaroon_file
-            .clone()
-            .unwrap_or_else(|| default_macaroon_file(&self.network))
+        match self.invoice_macaroon_file {
+            Some(_) => self.invoice_macaroon_file.clone().unwrap(),
+            None => self
+                .admin_macaroon_file
+                .clone()
+                .unwrap_or_else(|| default_macaroon_file(&self.network)),
+        }
```

Comment:
can we do a check that both invoice and admin aren't set before, could imagine people being confused if they set both

Reply:
I used an if with `panic!` now